### ISSUE TITLE
Forge Essentials Multi World causes Console Spam

### ISSUE THREAD

**Issue by Price-MA-Butta** [ForgeEssentials/ForgeEssentials#2416]
I have switched between many versions of Forge Essentials from 1146 to 1357. Whenever I create a multi-world no matter what id I use, what I name it, or what world type I use, it spams my console. Whenever it is deleted the world, it stops right away like nothing happened. This started (I think) when I copied and pasted a structure with flans guns and golden bags of holding from my main world to the multi-word. I did it once before with other similar items but I did //undo and it stopped. I can't undo it or remove the structure to fix it. This happens on new multi-worlds as well. I even deleted the files with FileZilla we well. I can't update because of AM2 so I'm looking for another idea or any suggestions on what the spam means. Here is the spam message.
[spam.txt](https://github.com/ForgeEssentials/ForgeEssentials/files/3052154/spam.txt)

**Comment by Zethalion** [comment]
https://github.com/ForgeEssentials/ForgeEssentials#reporting-bugs

**Comment by Price-MA-Butta** [comment]
[modlist.txt](https://github.com/ForgeEssentials/ForgeEssentials/files/3052162/modlist.txt)

This is the mod list and spam.txt is the log but only written once rather than spammed. I am using the latest forge essentials.

**Comment by Zethalion** [comment]
We need more

**Comment by Zethalion** [comment]
And use pastebin or gist next time

**Comment by Zethalion** [comment]
Can you recreate the issue with FE and the mod that provides the dimension you create only? That will really help us narrow down what is happening.

**Comment by Price-MA-Butta** [comment]
Tldr: I recreated the issue with FE on another server and FE creates the dimensions. Sorry if I miss understood. Testing a conflict and will get back to you.

Long version:
I when to an old host that was frozen before I made any multi-worlds or did any world editing. So this is basically a copy of the server just from two weeks ago. I thought there was not originally an error but I guess I never noticed. I'm going to test if it is Empires mod that is causing it because I just remembered that it was weird to install so it was active and inactive as I tested with it and I tend to jump around a lot with my tasks. It gives errors that are not very helpful and to me, this error is not very helpful. 

So maybe that is the missing link. I'll try to change the perms so it doesn't interact with the multi-world. The mod is a towny rip off as a mod and not a plugin so this could really just be an empires error. I'll update you after I do the tests from above.
